wordpress-mobile/WordPress-Android
# Changes: The tab title to Menu from Site Menu

Fixes #16305 

This PR changes the title of the tab from `Site menu` to `Menu` as per [this reference](pbArwn-4jn). 

![Screenshot_20220412_192436](https://user-images.githubusercontent.com/17463767/162980405-53cfd562-bbe4-4538-a0de-154078bf957f.png)


To test:

Test 1 
- Go to My Site 
- Notice that the title of the tabs are `Home` and `Menu` 

Test 2
- Go to My Site -> App Settings -> Initial Screen 
- Notice the selections are `Home` and `Menu`

Test 3 
- Go to My Site screen with a Site having Quick Start in Progress 
- Set Initial screen as Home 
- Start any quick start(Visit Site, Explore Plans or Post Sharing) which has focus point in `Menu` tab
- Notice that the snackbar prompt says to click at the `Menu` tab

## Regression Notes
1. Potential unintended areas of impact
Tab title is not shown properly in 
- Initial screen settings 
- Quick start snack bar prompt 


2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing 

3. What automated tests I added (or what prevented me from doing so)
none 

PR submission checklist:
